packages/documentation/src/pages/architecture/unit-tests.mdx

```diff
@@ -0,0 +1,120 @@
+## How do unit tests work?
+
+#### Where they live
+- Tests can live anywhere in the `src` directory, but in general are located in a directory close to the code or application that they are written to test.
+- The `.unit.spec.js` file extension is used for unit tests, while `.e2e.spec.js` is used for end-to-end (E2E) tests.
+- Visual regression test config has its own directory under `src/platform/testing/visual-regression`
+- Test commands are specified in `package.json`, as well as in vets-website's `README.md`
```

Comment:
We can add the most common test commands `yarn run test:unit` and `yarn run test:e2e` and Add links to the package.json and readme files so they're easy to get to